UnitsNet/CompiledLambdas.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace UnitsNet
+{
+    /// <summary>
+    /// Compiled lambda expressions that can be invoked with generic run-time parameters.
+    /// </summary>
```

Comment:
We could mention that this is all about performance vs reflection-based techniques.